config/sync/block.block.adminfeedbackblock.yml

```diff
@@ -0,0 +1,30 @@
+uuid: d6ea7a6c-909a-4214-9a64-69e3cc967662
+langcode: en
+status: true
+dependencies:
+  module:
+    - admin_feedback
+    - ctools
+  theme:
+    - pineapple
+id: adminfeedbackblock
+theme: pineapple
+region: content
+weight: 0
+provider: null
+plugin: admin_feedback_block
+settings:
+  id: admin_feedback_block
+  label: 'Admin Feedback Block'
+  provider: admin_feedback
+  label_display: '0'
+visibility:
+  'entity_bundle:node':
+    id: 'entity_bundle:node'
+    bundles:
```

Comment:
Can you also try and add the block on the recipes and articles pages, imagine those pages to be high traffic and would be good to have feedback on them.

Reply:
sorry but it is on all pages?

Reply:
Please see this page: http://localhost:7777/recipes and equivalent of articles

Reply:
Oh I understand now. No problem.